beets/library.py

```diff
@@ -410,16 +410,19 @@ def _get(self, key):
             raise KeyError(key)
 
     def __getitem__(self, key):
-        """Get the value for a key. Certain unset values are remapped.
+        """Get the value for a key. `artist` and `albumartist`
+		are fallback values for each other when unmapped.
```

Comment:
Better wording: "when not set"?

Reply:
Yeah, good point—"when not set" is better.